_rules/menuitem-non-empty-name-m6b1q3.md

```diff
@@ -116,7 +116,7 @@ This element with the `menuitem` role has no [accessible name][] because it has
 
 #### Failed Example 2
 
-This element with the `menuitem` role has no [accessible name][] because it has no content or attribute that can provide it.
+This offscreen element with the `menuitem` role has no [accessible name][] because it has no content or attribute that can provide it.
```

Comment:
```suggestion
This off-screen element with the `menuitem` role has no [accessible name][] because it has no content or attribute that can provide it.
```